tasks/sync_repos.py

```diff
@@ -78,6 +83,12 @@ async def run_async(
                         await self.sync_repos(
                             db_session, git, owner, username, using_integration
                         )
+
+                self.sync_repos_languages(
```

Comment:
I was unsure if I should put this inside the try or after the try/catch, thoughts? 

If outside, it might not get called if unsuccessful, but if inside, I'm not sure if this prevents this task from finishing - I assume not sync we're not awaiting for it to finish?

Reply:
I don't think you need a try/catch for this.

Also yes. When you call `apply_async` you enqueue the task. That returns a task ID that you can use to wait around and get results, but we're not doing that.

Reply:
Sounds good, not awaiting for it makes sense to me